app/jobs/reindex_collections_job.rb

```diff
@@ -2,7 +2,7 @@
 
 class ReindexCollectionsJob < ApplicationJob
   def perform
-    Collection.find_each do |collection|
+    PcdmCollection.find_each do |collection|
```

Comment:
This needs to be change to use the config collection class. Possibly should also reference multiple models?